Chat box changes

Tim3Game opened the pull request:
Hello,
I saw that you accepted and later added features from the Issue #237. I took the liberty of reviewing it and saw a few things that cough my eye, so I changed them. I wanted to do it sooner myself, but sadly work didn't allowed me to. Hope you don't mind my changes.

**List of changes and additions:**
- replaced `isPlayer(ServerPlayerEntity player, String argument)` with `getPlayer(String argument)` - this allows to get the player (or null if not available) by the username or UUID (if it matches the RegEx for it) instead of looping thru the player list and checking for fitting criterias,
- changes to `appendPrefix(...)` - I changed it from a string to a TextComponent oriented function, this allows for formatting of the prefix itself,
- moved the range argument from the last position to the 3rd one (after the prefix argument), because from my point of view users gonna want to set the range more often the prefix, but that's only my view on it. If it doesn't fit you, you can change it back.
- added "error" messages when the username/uuid is wrong (not found) and when the bracket format is wrong (because from what I saw it expected 2 characters,
- removed the range argument from the "...ToPlayer()" functions, because the range was mostly meant when it sends to multiple players, so it can be set up in a way to send a message to all players in a specific range and not the whole server.

There are other small changes, but they mostly cosmetic code. If you not happy with something, feel free to change it.

Tim3Game committed:
Chat box changes
- added error messages for invalid player name/uuid, bracket format,
- added uuid check and removed player loop when sending message to player,
- replaced isPlayer() with getPlayer(),
- reworked appendPrefix() function,
- other code cosmetic changes.

Seniorendi commented:
> moved the range argument from the last position to the 3rd one (after the prefix argument), because from my point of view users gonna want to set the range more often the the prefix, but that's only why view on it. If it doesn't fit you, you can change it back.

The reason I didn't do that was to support older programs. Some older programs use the 3 parameter for the prefix.

Your PR looks good, I will merge it later the day or tomorrow 👍🏻

Seniorendi commented:
Ignore the checks. Build was successful. Only the upload does not work.

Tim3Game commented:
> The reason I didn't do that was to support older programs. Some older programs use the third parameter for the prefix.

Oh sorry, didn't knew that, because from the wiki the only third parameter was in `sendMessageToPlayer()` and there I removed the `range` argument all together, and in `sendMessage()` and `sendFormattedMessage()` I moved it after the prefix argument to not mess with backwards compatibility.
